examples/kubernetes-config/kubernetes-clustersubnet.json

```diff
@@ -14,7 +14,7 @@
     },
     "agentPoolProfiles": [
       {
-        "name": "agent",
+        "name": "agentpool1",
```

Comment:
That was a previous CR feedback by @dmitsh. Instead of creating a new directory for every little KubernetesConfig knob we introduce, we are grouping them under one directory.